Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,6 +1,42 @@
-function setAlarm() {}
+function setAlarm() {
+  const inputTime = document.getElementById("alarmSet").value;
+  const timeDisplay = document.getElementById("timeRemaining");
+  const alarmInput = document.getElementById("alarmSet");
+  const setAlarmButton = document.getElementById("set");
+  let countDownInterval;
+  const seconds = alarmInput.value;
```

Comment:
What's the data type of `alarm.Input.value` ? Is that the data type you expect to use in your code?

Reply:
data type of alarmInput.value is string. I amended it to Number(alarmInput.value) to convert it to a number.
Thanks